src/plugins/tabs/components/tabs.vue

```diff
@@ -123,6 +123,7 @@ export default {
             this.composition.on('remove', this.removeItem);
             this.composition.on('reorder', this.onReorder);
             this.composition.load().then(() => {
+                this.setCurrentTab = true;
```

Comment:
This will also catch the case where the composition is empty and someone adds stuff to it after the view is mounted right?
OR if there is an error loading the composition?

Reply:
good catch. i need to set `this.setCurrentTab = false` after setting current tab in this block.